### PR TITLE
chore: add more data-testids for borrower profile page

### PR DESCRIPTION
**Pull request by davidskiva**
These should be all the data-testids need.

### CODE REVIEW

File: src/components/BorrowerProfile/DescriptionListItem.vue [hunk 1]
```diff
@@ -1,11 +1,15 @@
 <template>
 	<div class="tw-flex tw-mb-1.5">
 		<dt class="tw-flex-1 tw-mb-0">
-			<button @click="$emit('show-definition')" class="tw-underline tw-font-book">
+			<button @click="$emit('show-definition')" class="tw-underline tw-font-book"
+				data-testid="bp-details-lightbox-btn"
```
Comment (davidskiva) [inline src/components/BorrowerProfile/DescriptionListItem.vue:5]: These would need to be used in conjunction with the parent in either LoanDetails.vue, FieldPartnerDetails.vue and TrusteeDetails.vue.

File: src/components/BorrowerProfile/BorrowerBusinessDetails.vue [hunk 2]
```diff
@@ -36,6 +39,7 @@
 					tw-inline-flex tw-items-center tw-justify-center
 					tw-mt-2 tw-mr-2
 					tw-h-6 tw-w-6"
+			:data-testid="`bp-direct-business-social-links-${index}`"
```
Comment (mcstover) [inline src/components/BorrowerProfile/BorrowerBusinessDetails.vue:42]: If you wanted some additional specificity you could use `link.type` in place of `index` here to show "facebook" or "twitter" etc...

File: src/components/BorrowerProfile/DescriptionListItem.vue (hunk 1 above)
Comment (mcstover) [reply src/components/BorrowerProfile/DescriptionListItem.vue:5]: Correct and likely some "term" matching if you want to click any specific item...

Comment (mcstover) [reply src/components/BorrowerProfile/DescriptionListItem.vue:5]: Oh I see you have some specific wrappers below too!

**Review by mcstover — APPROVED** [review]

File: src/components/BorrowerProfile/BorrowerBusinessDetails.vue (hunk 2 above)
Comment (davidskiva) [reply src/components/BorrowerProfile/BorrowerBusinessDetails.vue:42]: I did like the idea of using link.type, but after careful consideration I think using index is best.  It is just easier to iterate through a loop to see which, if any, social link exist.  We can either get the href attribute for the element or alt attribute for the child svg to determine which site it is.

**Pull request merged**